### DISCUSSION TITLE
Table Partitioning support

### DISCUSSION THREAD

**Discussion by serg-kritsa** [sequelize/sequelize#16433] [Help & Questions] [answered]
Hi all.
Could someone explain does Sequelize support partitioning that implemented in Postgres (https://www.postgresql.org/docs/current/ddl-partitioning.html) out of the box?
Thanks.

**Comment by WikiRik** [comment, marked as answer]
Sequelize does not support partitioning at this moment, see https://github.com/sequelize/sequelize/issues/7673 for the feature request. There is a write up in there that can be further worked on by the community. The maintainers do not have this feature planned for the coming time
